flask_collect/storage/file.py

```diff
@@ -33,6 +33,8 @@ def run(self):
                 self.log("{0} already copied".format(destination))
             elif not op.exists(destination) or \
                     op.getmtime(destination) < op.getmtime(f):
+                if op.exists(destination):
```

Comment:
maybe it could worth it to optimize `op.exists` calls in `elif` branch.

### Example:
```python
else:
    dest_exists = op.exists(destination)
    if not dest_exists or \
            op.getmtime(destination) < op.getmtime(f):
        if dest_exists:
            remove(destination)
        ...
```
*if can be further improved*